RecoTracker/TkHitPairs/src/HitPairGeneratorFromLayerPair.cc

```diff
@@ -152,6 +152,8 @@ void HitPairGeneratorFromLayerPair::doublets(const TrackingRegion& region,
     for (int j = 0; j < 3; j += 2) {
       auto b = innerRange[j];
       auto e = innerRange[j + 1];
+      if (e == b)
```

Comment:
Isn't ```if (e <= b)``` even safer?

Reply:
If b > e then the whole algorithm would be busted, wouldn't it?